### ISSUE TITLE
missing executable vesion

### ISSUE THREAD

**Issue by marcgp** [cxpsemea/cxsast_custom_reporting#3]
**Describe the bug**
the executable released can not determine the version.

**To Reproduce**
Please provide detailed steps to reproduce the error
1. Download proper version
2. Extract zip file
4. execute binary
5. see error as follows
```
ConfigurationError: CO_MISSING_VERSION: Could not determine application version
    at getVersion (C:\snapshot\cxsast_custom_reporting\build\services\ConfigurationService\ConfigurationServiceImpl.js)
    at new ConfigurationServiceImpl (C:\snapshot\cxsast_custom_reporting\build\services\ConfigurationService\ConfigurationServiceImpl.js)
    at Function.getConfig (C:\snapshot\cxsast_custom_reporting\build\services\ConfigurationService\ConfigurationService.js)
    at Object.<anonymous> (C:\snapshot\cxsast_custom_reporting\build\services\LoggerService\LoggerService.js)
    at Module._compile (pkg/prelude/bootstrap.js:1261:22)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:993:10)
    at Module.load (internal/modules/cjs/loader.js:813:32)
    at Function.Module._load (internal/modules/cjs/loader.js:725:14)
    at Module.require (internal/modules/cjs/loader.js:850:19)
    at Module.require (pkg/prelude/bootstrap.js:1166:31) {
  code: 'CO_MISSING_VERSION',
  name: 'ConfigurationError'
}
```` 

**Expected behavior**
the application should initialize without problems

**Comment by marcgp** [comment]
:information_source::information_source: **This issue has been fixed in [Release v1.0.0-next.2](https://github.com/cxpsemea/cxsast_custom_reporting/releases/tag/v1.0.0-next.2)**

**Comment by marcgp** [comment]
:information_source::information_source: **This issue has been fixed in [Release v1.0.0](https://github.com/cxpsemea/cxsast_custom_reporting/releases/tag/v1.0.0)**
